src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/database/OppfolgingstilfelleDAO.kt

```diff
@@ -49,23 +49,29 @@ class OppfolgingstilfelleDAO @Inject constructor(
             personOppfolgingstilfelleRowMapper
         )
     }
-
-    fun update(oversikthendelsetilfelle: KOversikthendelsetilfelle) {
+    fun update(
+        fnr: Fodselsnummer,
+        kafkaOppfolgingstilfelle: KafkaOppfolgingstilfelle,
```

Comment:
Har du sett på muligheten for å inkapsulere feltene man trenger i en dataklasse som man kan sende inn til `update`og `create`?

Jeg synes kanskje det er litt uheldig at teknologivalg ("Kafka") og den dataklassen til den spesifikke køen (KafkaOppfolgingstilfelle) blør inn i DAO'en. Jeg ser at det allerede ligger en dataklasse som heter PersonVirksomhetOppfolgingstilfelle som inneholder de nødvendige feltene. Kanskje den kan brukes, evt opprette en annen?

Da får man en finere signatur `fun update(personVirksomhetOppfolgingstilfelle: PersonVirksomhetOppfolgingstilfelle)`, og man slipper mange av endringene i testene.

Reply:
Prøver det 👍